config.schema.json

```diff
@@ -67,6 +67,16 @@
                             "type": "boolean"
                         }
                     }
+                },
+                "warn": {
+                    "description": "Options for the warn command",
+                    "type": "object",
+                    "properties": {
+                        "mysql": {
+                            "description": "Toggles if a MySQL database should be used",
+                            "type": "boolean"
+                        }
+                    }
```

Comment:
I just noticed - what's up with this? Warn doesn't its own config (`config.commands.warn` doesn't exist) and `config.misc.mysql` has already been implemented

Reply:
I am not sure, I think it might be a relic from the past when warn existed as the only mysql function.

Reply:
You can remove it, as having it there would present `config.commands.warn` as something you can create even though it doesn't do anything

Reply:
I have class starting so this might never happen lol